frontend/app/views/spree/products/_taxons.html.erb

```diff
@@ -1,12 +1,10 @@
-<% if !@product.taxons.blank? %>
-  <div id="taxon-crumbs" data-hook class=" five ">
-    <h3 class="product-section-title"><%= Spree.t(:look_for_similar_items) %></h3>
-    <div data-hook="product_taxons">
-      <ul class="list-group" id="similar_items_by_taxon" data-hook>
-        <% @product.taxons.each do |taxon| %>
-          <li class="list-group-item"><%= link_to taxon.name, seo_url(taxon) %></li>
-        <% end %>
-      </ul>
-    </div>
+<% if @product.taxons.present? %>
+  <h3 class="product-section-title"><%= Spree.t(:look_for_similar_items) %></h3>
+  <div data-hook>
```

Comment:
Well I think just the ul below is fine so that div can go I think since you've moved the hook out to the other div#taxon-crumbs.